### ISSUE TITLE
Whitenoise:2.0 fails to flag a variable as unused if the only use is a 'set' statement in a report writer section.

### ISSUE THREAD

**Issue by feckertson** [cerner/ccl-testing#10]

**Comment by feckertson** [comment]
var0 is inappropriately not flagged as an unused variable
```
drop program prog go
create program prog 
    declare var0 = vc
    select into nl from dual detail
        var0 = 'HW'
    with nocounter
end go
```

**Comment by feckertson** [comment]
fixed in #11
